.gitignore

```diff
@@ -1,2 +1,3 @@
 /.output

```

Comment:
this file not needed right

Reply:
this file doesn't needs to be in the github so skipped it by declaring in .gitignore file